### ISSUE TITLE
spacewalk-report error: ImportError: cannot import name 'six'

### ISSUE THREAD

**Issue by kurzandras** [uyuni-project/uyuni#7344]
### Problem description

spacewalk-report can not be executed:

> spacewalk-report
Traceback (most recent call last):
  File "/usr/bin/spacewalk-report", line 31, in <module>
    from salt.ext import six
ImportError: cannot import name 'six'
[ERROR   ] An un-handled exception was caught by Salt's global exception handler:
ImportError: cannot import name 'six'
Traceback (most recent call last):
  File "/usr/bin/spacewalk-report", line 31, in <module>
    from salt.ext import six
ImportError: cannot import name 'six'

### Steps to reproduce

1. execute spacewalk-report on openSUSE Leap 15.4
2.
3.
...

### Uyuni version

```bash
Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : uyuni-server-stable
Name           : Uyuni-Server-release
Version        : 2023.04-220400.204.2.uyuni2
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes (automatically)
Status         : up-to-date
Source package : Uyuni-Server-release-2023.04-220400.204.2.uyuni2.src
Summary        : Uyuni Server
Description    :
    Uyuni lets you efficiently manage physical, virtual,
    and cloud-based Linux systems. It provides automated and cost-effective
    configuration and software management, asset management, and system
    provisioning.
```

### Useful logs

```bash
No related entries under/var/log/rhn
```

**Comment by kurzandras** [comment]
must be related to: https://www.suse.com/support/kb/doc/?id=000021130

https://ptf.suse.com/f2cf38b50ed714a8409693060195b235/susemanager-4.3/26373/x86_64/20230704?_gl=1*6ri9jp*_ga*MTQ3MTEzNTI1MS4xNjY2MjYwMjYw*_ga_JEVBS2XFKK*MTY5MDk4NTMyOC42MS4xLjE2OTA5ODU3NzIuNTcuMC4w

**Comment by avshiliaev** [comment]
Hey @kurzandras 
thanks for the report. The issue is indeed related to what you've linked. It's already fixed and the stable release (2023.08) is coming soon.

**Comment by kurzandras** [comment]
Thank you!
